apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/SyncOnboardingCompanion.tsx

```diff
@@ -479,12 +481,25 @@ const SyncOnboardingCompanion: React.FC<SyncOnboardingCompanionProps> = ({
     };
   }, [device, allowedError, handleDesyncTimerRunsOut, desyncTimeout]);
 
+  const recoverServices = useFeature("protectServicesDesktop");
+  const recoverStoreId = recoverServices?.params?.protectId ?? "";
+
+  const [recoverOnboardingStatus, setRecoverOnboardingStatus] = useState<boolean>();
+
+  const getRecoverOnboardingStatus = useCallback(async () => {
```

Comment:
I don't think you need the async
```suggestion
  const getRecoverOnboardingStatus = useCallback(() => {
```